api/v1/installation_types.go

```diff
@@ -660,3 +665,18 @@ type CertificateManagement struct {
 	// +optional
 	SignatureAlgorithm string `json:"signatureAlgorithm,omitempty"`
 }
+
+// ConditionTypes describes Installation CR's status condition type.
+// One of: Ready, Degraded, Progressing.
+type ConditionType string
+
+const (
+	// ConditionTypeReady indicates that the component is healthy.
+	ConditionTypeReady ConditionType = "Ready"
+
+	// ConditionTypeDegraded indicated that the component is not operating as desired and user action is required..
```

Comment:
nit: There are two periods at the end of the sentence.

Reply:
Done